### PR TITLE
Workaround archlinux jq failure

### PR DESCRIPTION
**Pull request by jayvdb**
Fixes https://github.com/taiki-e/install-action/issues/521

### CODE REVIEW

**Comment by jayvdb** [comment]
@taiki-e @NobodyXu this is green now

File: manifests/dprint.json [hunk 1]
```diff
@@ -25,10 +25,27 @@
   },
   "license_markdown": "[MIT](https://github.com/dprint/dprint/blob/main/LICENSE)",
   "latest": {
-    "version": "0.46.1"
+    "version": "0.46.2"
```
Comment (NobodyXu) [inline manifests/dprint.json:28]: The update of dprint should be in a separate PR, IMO we should wait for the GHA to auto-create and merge it, then remove this commit.

**Review by NobodyXu — CHANGES_REQUESTED** [review]

**Comment by jayvdb** [comment]
That is a catch-22 - the scheduled CI failed because of this https://github.com/taiki-e/install-action/actions/runs/9433528539

**Comment by NobodyXu** [comment]
So we need this PR to merge for the CI to run and auto-merge.

I see, this PR LGTM.

**Comment by NobodyXu** [comment]
There's still a problem of if it will show up in the commit history.

@jayvdb Could you clean up the commit history a bit?

I think doing a rebase merge might make more sense, to preserve the commit history.

cc @taiki-e What do you think?

**Comment by jayvdb** [comment]
I've updated the commit message of the middle commit.
The other two commits are different things, hence they are separate commits.

**Comment by jayvdb** [comment]
Green again

**Review by taiki-e — APPROVED** [review]
Thanks!
